tests/User-test.js

```diff
@@ -1,18 +1,186 @@
-// const chai = require('chai');
-// const expect = chai.expect;
-//
-// const Recipe = require('../src/Recipe');
+const chai = require('chai');
+const expect = chai.expect;
+
+const Recipe = require('../src/Recipe');
```

Comment:
Both of the files are capped by lowercase, think you might have just missed this one beings as the other classes have the right path.